### PR TITLE
Issue #858 - Improve frog control using touch/click in FoodChain activity

### PR DESCRIPTION
**Pull request by Keisuki**
- Fixes an issue where the mouse position would be incorrectly calculated when deciding which direction to move the frog
- Allows clicks in an area around the game area to still move the frog

### CODE REVIEW

**Comment by Keisuki** [comment]
I don't know how to link my pull request with the issue, so I'll just post the link here:
https://github.com/llaske/sugarizer/pull/860

File: activities/FoodChain.activity/playgame.js [hunk 1]
```diff
@@ -255,7 +255,14 @@ enyo.kind({
 	// Show direction to frog using click on board
 	clickToMove: function(s, e) {
 		// Compute direction comparing click with frog position
-		var dx = e.clientX-this.frog.getX(), dy = e.clientY-this.frog.getY(); 
+
+	  // Calculate the position of the mouse in world coordinates from viewport coordinates
```
Comment (llaske) [inline activities/FoodChain.activity/playgame.js:259]: Indentation is incorrect on this line.

**Comment by llaske** [comment]
Good job. Control is far better now.
Just an issue with scrollbar on some resolutions: scrollbar appear/disappear.

Peek 27-09-2020 10-42

Related to issue [858](url) (that's how you could link your PR to the issue)

**Comment by Keisuki** [comment]
What resolution does this appear on?  Having trouble replicating it.  
I'll fix the indentation issue tomorrow, don't know how I missed that.

File: activities/FoodChain.activity/playgame.js (hunk 1 above)
Comment (Keisuki) [reply activities/FoodChain.activity/playgame.js]: A tabs vs spaces issue.  I've fixed that now.

**Comment by Keisuki** [comment]
> 
> 
> Good job. Control is far better now.
> Just an issue with scrollbar on some resolutions: scrollbar appear/disappear.
> 
> Peek 27-09-2020 10-42
> 
> Related to issue [858](url) (that's how you could link your PR to the issue)

I believe i have fixed the scrollbar issue now.  Can you confirm?

**Comment by llaske** [comment]
Cool. It's fixed now.
Thanks for your contribution.

**Pull request merged**